js/Course.js

```diff
@@ -15,12 +15,12 @@ function Course(quarter, name, nameID, gt, units,
                 days, hour1, min1, hour2, min2, bldg, room) {
 
     //General information
-    this.quarter = quarter; //written as a string in the format of "season####"
+    this.quarter = (quarter === "" ? "None provided" : quarter); //written as a string in the format of "season####"
```

Comment:
But `===` tests for absolute equality. If you had two empty strings that occupied different locations in memory, then `quarter === ""` would be `false`.